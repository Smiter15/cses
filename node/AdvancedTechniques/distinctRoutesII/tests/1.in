13 15 2
1 12
1 2
12 3
2 4
2 5
4 11
5 13
3 6
3 5
6 7
7 8
8 9
9 10
10 11
11 13
