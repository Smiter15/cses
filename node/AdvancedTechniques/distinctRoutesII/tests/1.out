8
5
1 12 3 5 13 
5
1 2 4 11 13 
